.github/workflows/dogfood.yaml

```
@@ -51,22 +51,30 @@ jobs:
           tags: "codercom/oss-dogfood:${{ steps.docker-tag-name.outputs.tag }},codercom/oss-dogfood:latest"
           cache-from: type=registry,ref=codercom/oss-dogfood:latest
           cache-to: type=inline
+
   deploy_template:
     needs: deploy_image
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+
       - name: Get short commit SHA
         id: vars
         run: echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+
+      - name: Get Latest Commit Title
+        id: message
+        run: echo "pr_title==$(git log --format=%s -n 1 ${{ github.sha }})" >> $GITHUB_OUTPUT
+
       - name: "Get latest Coder binary from the server"
         run: |
           curl -fsSL "https://dev.coder.com/bin/coder-linux-amd64" -o "./coder"
           chmod +x "./coder"
+
       - name: "Push template"
         run: |
-          ./coder templates push $CODER_TEMPLATE_NAME --directory $CODER_TEMPLATE_DIR --yes --name=$CODER_TEMPLATE_VERSION
+          ./coder templates push $CODER_TEMPLATE_NAME --directory $CODER_TEMPLATE_DIR --yes --name=$CODER_TEMPLATE_VERSION --message=$CODER_TEMPLATE_MESSAGE
```

```suggestion
          ./coder templates push $CODER_TEMPLATE_NAME --directory $CODER_TEMPLATE_DIR --yes --name=$CODER_TEMPLATE_VERSION --message="$CODER_TEMPLATE_MESSAGE"
```
All args that may contain spaces need quoting, we could do it for all just as well (safer) but technically message is the only one with spaces currently.

---

Thank you :)